packages/__docs__/globals.js

```diff
@@ -53,11 +53,11 @@ const globals = {
   ...Components,
   debounce,
   moment,
-  locales: moment.locales(),
-  avatarSquare,
-  avatarPortrait,
+  locales: moment.locales(), // unused TODO double check, should be used DateInput
+  avatarSquare, // image
+  avatarPortrait, // image
   DateTime,
-  iconExample,
+  iconExample, // image
```

Comment:
you should remove these comments. Also locales seems to do nothing, I think it can be deleted. According to moment docs, it just returns all the locales, no idea what it was doing here. https://momentjs.com/docs/#/i18n/getting-locale/